.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Isn't it `bundle`?

Reply:
As this PR enables GitHub Action on this project, we want to enable Dependabot for the GitHub Action dependencies.

And as this project is a gem we don't have a `Gemfile.lock`. Constraints seems to be declared in a `gemspec` file (https://github.com/Scalingo/omniauth-scalingo/blob/master/omniauth-scalingo.gemspec#L19-L24) and I'm not sure if it makes it possible to enable Dependabot also for the Ruby dependencies.